make/test.mk

```diff
@@ -106,23 +106,23 @@ endif
 # e2e test in dev mode
 #-------------------------------------------------------------------------------
 
-.PHONY: e2e-cleanup
+.PHONY: e2e-cleanup-local
 ## Create a namespace used in e2e tests
-e2e-cleanup: get-test-namespace
+e2e-cleanup-local: get-test-namespace
 	$(Q)-oc login -u system:admin
 	$(Q)-oc delete -f ./deploy/crds/devconsole_v1alpha1_component_crd.yaml
 	$(Q)-oc delete -f ./deploy/service_account.yaml --namespace $(TEST_NAMESPACE)
 	$(Q)-oc delete -f ./deploy/role.yaml --namespace $(TEST_NAMESPACE)
 	$(Q)-oc delete -f ./deploy/test/role_binding_test.yaml --namespace $(TEST_NAMESPACE)
 	$(Q)-oc delete -f ./deploy/test/operator_test.yaml --namespace $(TEST_NAMESPACE)
 
-.PHONY: e2e-setup
+.PHONY: e2e-setup-local
 ## Create a namespace used in e2e tests
-e2e-setup: e2e-cleanup
+e2e-setup-local: e2e-cleanup-local
 	$(Q)-oc new-project $(TEST_NAMESPACE)
 
 .PHONY: build-image-local
-build-image-local: e2e-setup
+build-image-local: e2e-setup-local
```

Comment:
@baijum, As we need `build-image-local` too frequently for develop and test phase, I would avoid cleaning up resources for each docker build, otherwise to test some change I need to wait to clean up all openshift resources which adds more than few seconds.

Reply:
@dipak-pawar We can fix that in a separate PR.  It might require more discussion. For now, let me clear this name collision.

Reply:
but don't we need a clean slate before any test?